tests/dummy/lib/ember-chat/addon/views/application.js

```diff
@@ -1,20 +1,5 @@
-import Ember from 'ember';
+import Component from '@ember/component';
```

Comment:
I think we can remove this file, we never look up `views:application` as of Ember 2.x...